config/routes.rb

```diff
@@ -0,0 +1,24 @@
+Rails.application.routes.draw do
+  devise_for :users, :controllers => {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  
+  authenticated :user do
+    root to: "groups#index", as: :authenticated_root
+  end
+  
+  unauthenticated :user do
+    root "splash#index"
+  end
+  # Defines the root path route ("/")
+  # root "articles#index"
+  
+  resources :groups do
+    resources :products
+  end
```

Comment:
- [ ] It will be great to only develop only the routes that are used in the application. For example, the `delete, edit, update` routes are not used in the application. You can use `:only` to generate only the routes used in the application.